libs/blocks/media/media.js

```diff
@@ -18,6 +18,14 @@ import { decorateBlockBg, decorateBlockText, getBlockSize } from '../../utils/de
 import { decorateBlockAnalytics } from '../../martech/attributes.js';
 import { createTag } from '../../utils/utils.js';
 
+const decorateVideo = (container) => {
+  const link = container.querySelector('a[href$=".mp4"]');
+
+  container.innerHTML = `<video preload="metadata" playsinline autoplay muted loop>
+    <source src="${link.href}" type="video/mp4" />
+  </video>`;
+};
+
```

Comment:
I agree, It might be better to pu this in a shared function `decorateVideoAsset()` or something in `utils/decorate.js` and then include it as a video asset decorator in media and marquee. General rule is if a util is needed across multiple blocks, create an import func

Reply:
+1 for placing in decorate JS. 

I wonder if we also wanted to take some of the authoring options for autoplay that might be useful here that are in `video/video.js`?

Reply:
I'll remove the decorateVideo function off the media.js file and update video.js to add on attributes to help accessibility, i.e. `aria-hidden="true"`